.github/workflows/build.yml

```diff
@@ -296,6 +296,16 @@ jobs:
         uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
         if: always() && env.coverage == 'true'
```

Comment:
we should change it too
```suggestion
         if: (success() || failure()) && env.coverage == 'true'
```

Reply:
But if codecov servers are unavailabe for quite a while, we're in trouble

Reply:
Ah, I see, we're in trouble now :)